exercises/practice/killer-sudoku-helper/.docs/instructions.md

```diff
@@ -0,0 +1,47 @@
+# Instructions
+
+A friend of yours is learning how to solve Killer Sudokus but struggling to figure out which digits can go in a cage.
```

Comment:
```suggestion
A friend of yours is learning how to solve Killer Sudokus but struggling to figure out which digits can go in a grid.
```

I've never heard of "cage" in this context, and [the wikipeida page](https://en.wikipedia.org/wiki/Sudoku) doesn't use the term. I've always know them (and other similar things) to be called "grids".

Reply:
Right, I understand what a cage is now I think, but I was really confused reading this without knowing that, so I'd recommend defining "cage" before you use it. So ignore my previous suggestion, but I think we need to say in the first sentence that the rules are defined below (so someone doesn't do what I did and go googling), and then define the term "cage" in the rules.

```suggestion
A friend of yours is learning how to solve Killer Sudokus (rules below) but struggling to figure out which digits can go in a cage.
```

Reply:
Makes sense, thanks for the improvement :)